CONTRIBUTING.md

```diff
@@ -14,11 +14,11 @@ This guide will help you get started and outline some things you should know whe
 
 * install [Go](https://golang.org)
   * on macOS via `brew install go`
-* install [Glide](https://github.com/Masterminds/glide) (package manager for Go)
-  * on macOS: `brew install glide`
+  * on Windows via the [official Go installer](https://golang.org/dl)
 * install [Ruby 2.3.3](https://www.ruby-lang.org/en/documentation/installation) to run the feature tests
   * prefer install with [rbenv](https://github.com/sstephenson/rbenv)
-* run `gem install bundler`
+  * run `gem install bundler`
+* run `bin/setup` to install the dependencies
```

Comment:
This was already on line `29`. Maybe we need the go path setup installing anything

Reply:
I have cleaned this up.